community/examples/ml-gke.yaml

```diff
@@ -19,6 +19,7 @@ vars:
   project_id:  ## Set GCP Project ID Here ##
   deployment_name: ml-01
   region: us-central1
+  zones: [us-central1-c]
```

Comment:
Did you mean for this to make it in the final version, or is it an artifact from testing?

Reply:
I can remove this as this was a change from a2 , when I was not able to get capacity , for g2 I probably don't need this .

Reply:
Turns out I was wrong, `nvidia-l4` is not available in all zones so hardcoded to `us-central1-c` where it is present. Will add a comment to blueprint to record this reason.